libs/vkd3d/device.c

```diff
@@ -4218,6 +4219,36 @@ static void vkd3d_null_rtas_allocation_cleanup(
     }
 }
 
+static void d3d12_device_open_kmt(struct d3d12_device *device)
+{
+#ifdef _WIN32
+    D3DKMT_OPENADAPTERFROMLUID open_adapter = {0};
+    open_adapter.AdapterLuid = device->adapter_luid;
+
+    if (!D3DKMTOpenAdapterFromLuid(&open_adapter))
```

Comment:
Is this actually checking for success? We tend to use SUCCEEDED() macros in these 0-is-success cases.

Reply:
SUCCEEDED is for HRESULT, this is returning a NTSTATUS, which also have success == 0 but otherwise are a different type and set of values.